src/styles/reset.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
use `normalize.scss` file
`normalize` - to set default values
`reset` - to reset the values